src/ipc/capnp/mining.capnp

```diff
@@ -33,6 +33,7 @@ interface BlockTemplate $Proxy.wrap("interfaces::BlockTemplate") {
     getWitnessCommitmentIndex @6 (context: Proxy.Context) -> (result: Int32);
     getCoinbaseMerklePath @7 (context: Proxy.Context) -> (result: List(Data));
     submitSolution@8 (context: Proxy.Context, version: UInt32, timestamp: UInt32, nonce: UInt32, coinbase :Data) -> (result: Bool);
+    destroy @9 (context :Proxy.Context) -> ();
```

Comment:
In commit "Add destroy to BlockTemplate schema" (5f7dc0dbe64eb38afc74e9c4cba489382ea373bd)

Would suggest making destroy the first method because it affect semantics of the object as a whole, and thats where I placed destroy in all the other interfaces. It is fine to keep the number @9 for it, or renumber if you prefer.

Reply:
I moved `destroy` to the top and renumbered.